src/pdfpc.vala

```diff
@@ -122,8 +122,11 @@ namespace pdfpc {
                 ref Options.external_script,
                 "Enable execution of an external script", "file"},
             {"size", 'Z', 0, OptionArg.STRING,
-                ref Options.size,
-                "Size of the presentation window (implies \"-w\")", "W:H"},
+                ref Options.presentation_size,
+                "Size of the presentation window (implies \"-w both\")", "W:H"},
+            {"pc-size", 'A', 0, OptionArg.STRING,
+                ref Options.presenter_size,
+                "Size of the presenter window (implies \"-w both\")", "W:H"},
```

Comment:
See above. Also, please update the man page.

Reply:
Felt rather weird writing the man page without stabilizing the behaviour beforehand, but I went ahead with a first draft.